app/controllers/requests_controller.rb

```diff
@@ -43,10 +42,22 @@ def update
     end
   end
 
+  def success
```

Comment:
@cbeer what is this empty method doing (and `def ineligible` on the next lines)?

Reply:
nevermind...just answered my own question